tests/test_maintel_offset_mtcs.py

```diff
@@ -0,0 +1,187 @@
+            dict(offset_xy=dict(x=10, y=30), offset_azel=dict(az=180, el=60)),
+            dict(offset_xy=dict(x=10, y=30), reset_offsets=dict()),
+        ]
+
+        self.remotes_needed = False
+        async with self.make_script():
+            for config in configs_bad:
+                with pytest.raises(salobj.ExpectedError):
+                    await self.configure_script(**config)
+
+                    assert self.state.state == ScriptState.CONFIGURE_FAILED
+
+    def assert_config(self, default_values, config):
+        configured_values = dict(
+            offset_azel=self.script.offset_azel,
+            offset_radec=self.script.offset_radec,
+            offset_xy=self.script.offset_xy,
+            offset_rot=self.script.offset_rot,
+            reset_offsets=self.script.reset_offsets,
+            absorb=self.script.absorb,
+            relative=self.script.relative,
+        )
+
+        for parameter in default_values:
+            with self.subTest(config=config, parameter=parameter):
+                assert (
+                    config.get(parameter, default_values.get(parameter))
+                    == configured_values[parameter]
+                )
+
+    async def test_offset_azel(self):
+        async with self.make_script():
+            config = dict(offset_azel=dict(az=10, el=-10), absorb=False, relative=False)
+
+            await self.configure_script(**config)
+
+            self.script.mtcs.offset_azel = unittest.mock.AsyncMock()
+
+            await self.script.run()
+
+            self.script.mtcs.offset_azel.assert_awaited_once_with(
+                az=10, el=-10, absorb=False, relative=False
+            )
+
+    async def test_offset_radec(self):
+        async with self.make_script():
+            config = dict(offset_radec=dict(ra=1, dec=-1))
+
+            await self.configure_script(**config)
+
+            self.script.mtcs.offset_radec = unittest.mock.AsyncMock()
+
+            await self.script.run()
+
+            self.script.mtcs.offset_radec.assert_awaited_once_with(
+                ra=1,
+                dec=-1,
+                absorb=False,
+                relative=True,
+            )
+
+    async def test_offset_xy(self):
+        async with self.make_script():
+            config = dict(offset_xy=dict(x=10, y=-10))
+
+            await self.configure_script(**config)
+
+            self.script.mtcs.offset_xy = unittest.mock.AsyncMock()
+
+            await self.script.run()
+
+            self.script.mtcs.offset_xy.assert_awaited_once_with(
+                x=10,
+                y=-10,
+                absorb=False,
+                relative=True,
+            )
+
+    async def test_offset_rot(self):
+        async with self.make_script():
+            config = dict(offset_rot=dict(rot=10))
+
+            await self.configure_script(**config)
+
+            self.script.mtcs.offset_rot = unittest.mock.AsyncMock()
+
+            await self.script.run()
+
```

Comment:
Make sure you assert that `self.script.mtcs.offset_rot` was awaited with the expected value.

Reply:
I think that's being done in L166 below, right?